storage/client/storage_client.bzl

```diff
@@ -0,0 +1,10 @@
+# DO NOT EDIT -- GENERATED CODE
```

Comment:
This is made by `create_bazel_config` I presume? It would be nice (in the future) to have that script output a path to the code that generates it as part of the comment, to direct the person reading this file where/how to update it, rather than submit a PR to modify it (manually), just to be told where the source is (again, manually).

Reply:
Ack.  Good idea.